lib/view_component/system_test_helpers.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module SystemTestHelpers
+    include TestHelpers
+    include Capybara::DSL
+
+    def page
+      Capybara.current_session
+    end
+
+    def visit_rendered_in_browser(component, **options)
+      layout = options[:layout] || false
+
+      opts = {
+        layout: layout,
+        locals: {
+          render_args: {
+            component: component,
+            template: "view_components/preview",
+            hide_preview_source: true
+          }
+        }
+      }
+      html = controller.render_to_string("view_components/preview", opts)
+
+      # Write to temporary file to contain fully rendered component
+      # within a browser
+      file = Tempfile.new(["rendered_#{component.class.name}", ".html"], "tmp/")
+      file.write(html)
+      file.rewind
+
+      # NOTE - not entirely sure how this would work
+      # given that the application may have their own capybara
+      # instance running
```

Comment:
Do you feel this comment is still accurate?

Reply:
Hmm, it shouldn't be, since I believe the system test suite should handle this and my original concern with the comment isn't relevant. I'll remove it